Anlab.Mvc/Controllers/OrderController.cs

```diff
@@ -231,8 +241,19 @@ public OrderController(ApplicationDbContext context, IOrderService orderService,
                 return RedirectToAction("Index");
             }
 
-            order.Status = OrderStatusCodes.Confirmed;
+            var allTests = await _orderService.PopulateTestItemModel(true);

```

Comment:
@laholstege this code is copied several times, and at the end it just essentially takes an order and returns an updated order -- maybe a combined method in the order service would save us from future copy errors?  `_orderService.UpdateTestsAndRecalculate(order)` or similar?